src/cli/tc.rs

```diff
@@ -54,51 +56,44 @@ pub fn run() -> Result<()> {
 }
 
 pub async fn init(file_path: &str) -> Result<()> {
+  let file_name = ".tailcallrc.graphql";
+  let file_name_yml = ".graphqlrc.yml";
   let tailcallrc: resource::Resource<str> = resource_str!("examples/.tailcallrc.graphql");
+  let tailcallrc_path = Path::new(file_path).join(file_name);
+  let graphqlrc = format!(
+    r#"|schema:
+       |- './{}'
+       |- './{}'
+    "#,
+    &file_name, file_name
```

Comment:
Since we don't create the user's `.graphql` file, we no longer need to declare it on `graphqlrc`.

```suggestion
  let graphqlrc = format!(
    r#"|schema:
       |- './{}'
    "#,
    &file_name
```

Reply:
fixed, good catch thank ypu